packages/cli/src/commands/hydrogen/init.test.ts

```diff
@@ -321,6 +322,116 @@ describe('init', () => {
         expect(output).toMatch(/Language:\s*JavaScript/);
       });
     });
+
+    it('creates functional classic Remix projects', async () => {
+      await inTemporaryDirectory(async (tmpDir) => {
+        await runInit({
+          path: tmpDir,
+          git: false,
+          language: 'ts',
+          template: 'classic-remix',
+          routes: true,
+          installDeps: true,
+        });
+
+        const templateFiles = await glob('**/*', {
+          cwd: getSkeletonSourceDir()
+            .replace('templates', 'examples')
+            .replace('skeleton', 'classic-remix'),
+          ignore: ['**/node_modules/**', '**/dist/**'],
+        });
+        const resultFiles = await glob('**/*', {cwd: tmpDir});
+        const nonAppFiles = templateFiles.filter(
+          (item) => !item.startsWith('app/'),
+        );
+
+        expect(resultFiles).toEqual(expect.arrayContaining(nonAppFiles));
+        expect(resultFiles).not.toContain('vite.config.ts');
+        expect(resultFiles).not.toContain('env.d.ts');
+
+        await expect(readFile(`${tmpDir}/package.json`)).resolves.toMatch(
+          `"name": "example-classic-remix"`,
+        );
+
+        // ---- DEV
```

Comment:
I've been thinking about this and would like to split this file in different ones. But I think this could still be here.
The ones I want to extract are the "build" and "dev" tests.
These should be in `build.test` and `dev.test` even if they use `runInit` to generate fixtures 🤔